code/script.js

```diff
@@ -1,9 +1,27 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
+const formSubmitButton = document.getElementById("send-btn");
+const formMessageBox = document.getElementById("name-input");
+const inputWrapper = document.getElementById("input-wrapper");
+
 
 // Global variables, if you need any, declared here
+let questionNumber = 1;
+let chatDelay = 800;
```

Comment:
If you do not want to override the value of your variable you should use `const` to create it 🤓